services/product-service/Dockerfile

```
@@ -1,21 +1,16 @@
-RUN ./mvnw dependency:go-offline -B
-COPY src src
-RUN ./mvnw clean package -DskipTests
+COPY pom.xml .
+COPY services/product-service/pom.xml services/product-service/
+COPY services/product-service/src services/product-service/src
+COPY shared shared
+RUN mvn -B -pl services/product-service -am install -DskipTests
 
-FROM eclipse-temurin:17-jre-jammy
+# Stage 2: Runtime
+FROM gcr.io/distroless/java17-debian12:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)